internal/datastore/memdb/readonly.go

```diff
@@ -306,6 +306,8 @@ func filterFuncForFilters(
 				return true
 			case len(relations) > 0 && !stringz.SliceContains(relations, tuple.subjectRelation):
 				return true
+			case optionalSubjectsFilter.OptionalCaveatName != "" && (tuple.caveat == nil || tuple.caveat.caveatName != optionalSubjectsFilter.OptionalCaveatName):
```

Comment:
Didn't we discuss the caveat didn't belong to  the subjects filter? Wasn't what I added in https://github.com/authzed/spicedb/pull/899/commits/114455fcc6f263ef5853fe52be4658de9eb960ca enough?

Reply:
It was removed in a later commit

Reply:
sorry, what was removed?